internal/certs/certs.go

```diff
@@ -142,3 +124,29 @@ func SelfSignedOrLetsEncryptCert(manager *autocert.Manager, serverName string) f
 		return &keypair, nil
 	}
 }
+
+func newCA() (*x509.Certificate, crypto.PrivateKey, error) {
```

Comment:
I moved this out into a function to make it easier to generate the certs that I checked into git.

I used:

```go
func TestCreateCerts(t *testing.T) {
	ca, pk, err := newCA()
	assert.NilError(t, err)

	certBytes, keyBytes, err := GenerateCertificate([]string{"127.0.0.1", "::1"}, ca, pk)

	ioutil.WriteFile("localhost.crt", certBytes, 0600)
	ioutil.WriteFile("localhost.key", keyBytes, 0600)
}
```

We could check this in somewhere as a way to re-generate them. They only last a year, so we'll have to do it again in 11 months.

Reply:
It would probably be fine to use a longer expiry. We hardcode to 1 year right now in the production code, so I just used the same value.